fixers.md

```diff
@@ -1,6 +1,21 @@
 # PHP Fixers
 
-We have many available fixers. Note that while header comment fixing is available in StyleCI, we do not consider it a "fixer" as isn't configured as a fixer. Header comment fixing has it's own dedicated config on the settings page for your repo.
+Our PHP fixing works by applying single-purposed transformations to files that fix specific issues.
+
+Depending upon the [preset](presets), a default set of fixers will be included.
+If you want more granular control, you can enable or disable specific fixers by adding them to your configuration:
+
+```diff
```

Comment:
The format should is `yaml` here. ;)

Reply:
I chose `diff` to show adding/removal of the lines. We can go with plain `yaml` too if you like that better.

Reply:
I'm not sure how this will actually render on our docs site. I'll try it out in the next few days, locally.

Reply:
Should i just change this to `yaml` so this PR can proceed?

Reply:
Changed it.